code/modules/antagonists/changeling/powers/fakedeath.dm

```diff
@@ -18,7 +18,6 @@
 		user.tod = STATION_TIME_TIMESTAMP("hh:mm:ss", world.time)
 	user.fakedeath("changeling") //play dead
 	user.update_stat()
-	user.update_canmove()
```

Comment:
sure this should be removed instead of having update_mobility?

Reply:
yes, update_stat calls it in carbon